src/styles/starts.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+
+    background-image: url(../images/star.svg);
```

Comment:
not fixed, at least note it:

![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/9bee25b5-485c-42bf-8f2c-e36a23b496f4)